src/pages/settings/client-portal/ClientPortal.tsx

```diff
@@ -34,16 +39,22 @@ export function ClientPortal() {
   ];
 
   const onSave = useHandleCompanySave();
+
   const onCancel = useDiscardChanges();
 
   return (
     <Settings
       title={t('client_portal')}
-      breadcrumbs={pages}
       docsLink="docs/advanced-settings/#client_portal"
+      breadcrumbs={pages}
       onSaveClick={onSave}
       onCancelClick={onCancel}
+      disableSaveButton={!proPlan() && !enterprisePlan() && isHosted()}
```

Comment:
Do we use this all the time? Can we extract it into hook?

Reply:
It's extracted to hook and applyed to all places where we use this condition for advance settings.